chunky/src/java/se/llbit/util/TaskTracker.java

```diff
@@ -114,6 +113,45 @@ public void update(String task, int target, int done, String eta) {
       this.eta = eta;
       update();
     }
+
+    /** Changes the task name and state. Calculates eta. */
+    public void update(String task, int target, int done, long startTime) {
```

Comment:
If it changes the task name, it's a new task. :thinking: The automatic ETA calculation is great, though! Could be used in `RenderManager.updateRenderProgress` too once this is merged.

Reply:
This is still important for usages where an existing task's name is changed. I don't intend on refactoring all the usages of task in this PR 🙂. However, I did change it to create a new task where relevant.